src/Helper/SplitStreet.php

```diff
@@ -77,6 +77,8 @@ public static function splitStreet(string $fullStreet, string $local, string $de
             foreach (self::NUMBER_SUFFIX_ABBREVIATION as $from => $to) {
                 $fullStreet = preg_replace("/(\d.*-?)[\s]$from/", '$1' . $to, $fullStreet);
             }
+
+            $fullStreet = trim(preg_replace('/(\r\n)|\n|\r/', ' ', $fullStreet));
```

Comment:
I think it's good to add a test to the split street tests.
Or you can of put this logic in a function. Than you can make a unit test for that function.